src/Cask/Cask.cs

```diff
@@ -429,24 +414,12 @@ private static void ValidateTimestamp(DateTimeOffset now)
 
     private static bool IsValidKeyLengthInChars(int length)
     {
-        if (length < MinKeyLengthInChars || length > MaxKeyLengthInChars || !Is4CharAligned(length))
-        {
-            return false;
-        }
-
-        // There is gap in valid key lengths between 384-bit and 512-bit keys.
-        return length <= Max384BitKeyLengthInChars || length >= Min512BitKeyLengthInChars;
+        return !(length < MinKeyLengthInChars || length > MaxKeyLengthInChars || !Is4CharAligned(length));
```

Comment:
I think it's a little easier to understand this way:

```suggestion
        return length >= MinKeyLengthInChars && length <= MaxKeyLengthInChars && Is4CharAligned(length);
```

Reply:
thank you for this transformation in propositional logic

Reply:
Don't thank me, thank De Morgan.